### PR TITLE
Cirrus: Fix e2e tests for "mount_rootless_test"

### PR DESCRIPTION
**Pull request by sstosh**
e2e `mount_rootless_test` did not load `podman binary path` successfully.
This PR fix this problem.

[It] podman unshare podman mount:
```
[+1596s] Running: ... unshare  mount <cid>
[+1596s] Error: exec: no command
[+1596s] output:
```

[It] podman unshare image podman mount:
```
[+1599s] Running: ... unshare  image mount quay.io/libpod/alpine:latest
[+1599s] Error: exec: no command
[+1599s] output:
```

Signed-off-by: Toshiki Sonoda <sonoda.toshiki@fujitsu.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM
@containers/podman-maintainers PTAL

**Comment by sstosh** [comment]
I updated the commit.
Could you check the new commit?

**Review by Luap99 — APPROVED** [review]
/lgtm
/hold

**Pull request merged**
